docs(router-protected): fix prop name (#40947)
expo/expo: docs/pages/router/advanced/protected.mdx

}
```

> Here, because the **index** screen is protected and the **guard** is **false**, the router redirects to the first available screen &mdash; **login**.

Reviewer: ```suggestion
In the above example, since the **index** screen is protected and the `guard` is **false**, the router redirects to the first available screen &mdash; **login**.
```

I'd like to update this sentence for clarity.
